environments/.stackhpc/hooks/pre.yml

```diff
@@ -15,3 +15,19 @@
         - "{{ lookup('env', 'APPLIANCES_ENVIRONMENT_ROOT') }}/inventory/hosts"
         - "{{ lookup('env', 'APPLIANCES_ENVIRONMENT_ROOT') }}/inventory/group_vars/all/secrets.yml"
         - "{{ lookup('env', 'APPLIANCES_ENVIRONMENT_ROOT') }}/inventory/group_vars/all/test_user.yml"
+
+- hosts: all:!builder
+  become: yes
+  gather_facts: false
+  tags: podman
+  tasks:
+    - name: Configure container image registry for unqualified searches to avoid docker.io ratelimits
```

Comment:
Does this need

`when: ci_cloud == 'ARCUS'`

?

Reply:
See
https://github.com/stackhpc/ansible-slurm-appliance/blob/main/environments/.stackhpc/inventory/group_vars/all/bastion.yml#L1
and
https://github.com/stackhpc/ansible-slurm-appliance/blob/main/.github/workflows/stackhpc.yml#L18

basically this gets set by a github actions variable to the cloud the CI is running on